src/boot/vue-intl.js

```diff
@@ -48,6 +50,29 @@ export default async ({ Vue, store }) => {
       fallbackValues[valueSearch[1].trim()] = undefined
     }
 
+    const contentEdition = store.state.content.contentEdition
+    const selectedEntry = store.state.content.selectedEntry
+
+    // emits the ICU format error only if content edition is enabled
+    // and if an entry is selected
+    if (contentEdition && selectedEntry) {
+      const { entry, field } = selectedEntry
+      const selectedEntryKey = `${entry}.${field}`
+
+      if (key === selectedEntryKey) {
+        try {
+          parser.parse(translation)
+        } catch (err) {
+          EventBus.$emit('postContentMessage', {
+            type: 'stelaceContentError',
+            entry,
+            field,
+            error: err.message
+          })
+        }
+      }
+    }
+
```

Comment:
Please see my comment in App.vue, suggesting you move this specific code from here.

This would allow to lazily import `intl-messageformat-parser` in App.vue.
“Only” 4kB gzipped but code will be cleaner too, in a single place if you use a mixin.

Reply:
Right.